i18n/strings_to_json_mapping.json

```diff
@@ -82,6 +83,8 @@
   "LOOKS_DELETE_LOOK": "${brick_delete_look}",
   "LOOKS_PAINT_NEW_LOOK": "${brick_paint_new_look} %1%2",
   "LOOKS_EDIT_LOOK": "${brick_edit_look}",
+  "SET_CAMERA_FOCUS": "${brick_focus_point} %1 ${brick_horizontal_and} %2 ${brick_vertical_flex} %3",
```

Comment:
Please fix arguments (%1%2 for brick_set_camera_focus_horizontal_input and %3%4 for brick_set_camera_focus_vertical_input) and add percent symbol (compare with [Catroid](https://github.com/Catrobat/Catroid/blob/1d2962b31c1483e1edc9b429460e77c0858cf4ae/catroid/src/main/res/layout/brick_set_camera_focus.xml))

Reply:
unfortunately this does not work as they are defined as following in the language.xml files
 <string name="brick_focus_point">Become focus point with</string>
    <string name="brick_horizontal_and">horizontal and</string>
    <string name="brick_vertical_flex">vertical flexibility</string>